src/changes.ts

```diff
@@ -36,6 +36,7 @@ const ISSUE_URL_REGEXP =
 
 const MERGE_COMMIT_REGEX = /Merge pull request #(\d+) from (.*)/;
 
+const COMMENT_REGEXP = /<!--(.*?)-->/s;
```

Comment:
Looks like this regex might not cut it if a comment spans over multiple line?

Reply:
It works fine, `s` mode makes `.` match newlines

![image](https://user-images.githubusercontent.com/2403652/181220617-7bee3263-6e0c-4dd0-8fe6-f74e8a536ee6.png)